src/App.tsx

```diff
@@ -72,6 +74,13 @@ const App = () => {
                   </ResetRequestProvider>
                 }
               />
+              <Route
+              path="/profile/cesta"
```

Comment:
```suggestion
              path="/profile/user/cesta"
```
cesta é inerente ao perfil de usuário